### ISSUE TITLE
[GLPI 10]  Dictionaries Software problem

### ISSUE THREAD

**Issue by Badkempachi** [glpi-project/glpi#10680]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0 rc-1

### Bug description

Hello, there is a problem when using the dictionary for “Software” . A group named "Perl package" has been created and a corresponding rule has been created for it, where the word that starts with "Perl-" will be placed in the group "Perl package". The rule is executed, and also packages are displayed in this group, but the number of installations is “0” although it should be “1” since 1 test virtual machine with Linux has been added. When I try to filter by the name "package" or "Perl" in the "Software" tab and I get the following message:
SQL Warnings: 1260: Row 942 was cut by GROUP_CONCAT() in query "SELECT DISTINCT `glpi_softwares`.`id` AS id, 'username_sa' AS currentuser, `glpi_softwares`.`entities_id`, `glpi_softwares`.`is_recursive`, `glpi_softwares`.`name` AS `ITEM_Software_1`, `glpi_softwares`.`id` AS `ITEM_Software_1_id`, `glpi_entities`.`completename` AS `ITEM_Software_80`, `glpi_manufacturers`.`name` AS `ITEM_Software_23`, GROUP_CONCAT( DISTINCT CONCAT(IFNULL(`glpi_softwareversions`.`name`, '__NULL__'), '$#$',`glpi_softwareversions`.`id`) ORDER BY `glpi_softwareversions`.`id` SEPARATOR '$$##$$' ) AS `ITEM_Software_5`, GROUP_CONCAT (DISTINCT CONCAT (IFNULL (` glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`name`, '__NULL__'), '$ # $', `glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id`) ORDER BY` glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id` SEPARATOR '$$##$$') AS `ITEM_Software_4`, COUNT(DISTINCT `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`id`) AS `ITEM_Software_72` , FLOOR (SUM ( `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`number`) * COUNT (DISTINCT` glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`id`) / COUNT ( `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`id`)) AS` ITEM_Software_163`, MIN ( `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`number `) AS `ITEM_Software_163_min` FROM `glpi_softwares`LEFT JOIN `glpi_entities` ON ​​(`glpi_softwares`.`entities_id` = `glpi_entities`.`id` )LEFT JOIN `glpi_manufacturers` ON ​​(`glpi_softwares`.`manufacturers_id` = `glpi_manufacturers `.`id`) LEFT JOIN` glpi_softwareversions` ON ​​( `glpi_softwares`.`id` =` glpi_softwareversions`.`softwares_id`) LEFT JOIN `glpi_operatingsystems` AS` glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7` ON ( `glpi_softwareversions`.`operatingsystems_id` =` glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7 `.`id` ) LEFT JOIN `glpi_items_softwareversions` AS `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9` ON (`glpi_softwarevers ions`.`id` = `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`softwareversions_id` AND` glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_deleted_item` = '0' AND `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_deleted` = '0' AND` glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_template_item` = '0' ) LEFT JOIN `glpi_softwarelicenses` AS` glpi_softwarelicenses_6427cab6934982da77f819e079502198` ON ( `glpi_softwares`.`id` =` glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`softwares_id` AND `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`is_template` = '0' AND ((` glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`expire` IS NULL) OR (`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`expire` > NOW())) ) WHERE `glpi_softwares`.`is_deleted` = 0 AND `glpi_softwares`.`is_template` = 0 AND ( ( (`glpi_softwares`.`name` LIKE '%package%' ) OR (`glpi_entities`.`completename` LIKE '%package%' ) OR (`glp i_manufacturers`.`name` LIKE '%package%' ) OR (`glpi_softwareversions`.`name` LIKE '%package%' ) OR (`glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`name` LIKE '%package%' )) ) GROUP BY ` glpi_softwares`.`id` ORDER BY `id` "

### Relevant log output

```shell
[2022-02-16 17:28:40] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined index: _system_category in /var/www/html/glpi/src/Software.php at line 787
  Backtrace :
  src/Software.php:851                               Software->addSoftware()
  src/RuleDictionnarySoftwareCollection.php:368      Software->addOrRestoreFromTrash()
  src/RuleDictionnarySoftwareCollection.php:284      RuleDictionnarySoftwareCollection->replayDictionnaryOnOneSoftware()
  src/RuleDictionnarySoftwareCollection.php:212      RuleDictionnarySoftwareCollection->replayDictionnaryOnSoftwaresByID()
  front/rule.common.php:105                          RuleDictionnarySoftwareCollection->replayRulesOnExistingDB()
  front/ruledictionnarysoftware.php:38               include()
  
[2022-02-16 17:38:06] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:08] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:10] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:663                                  Agent->requestAgent()
  ajax/agent.php:54                                  Agent->requestInventory()
  
[2022-02-16 17:38:12] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:12] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:15] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:663                                  Agent->requestAgent()
  ajax/agent.php:54                                  Agent->requestInventory()
  
[2022-02-16 17:38:15] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:663                                  Agent->requestAgent()
  ajax/agent.php:54                                  Agent->requestInventory()
  
[2022-02-16 17:38:21] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:22] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:22] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:23] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:663                                  Agent->requestAgent()
  ajax/agent.php:54                                  Agent->requestInventory()
  
[2022-02-16 17:38:26] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:663                                  Agent->requestAgent()
  ajax/agent.php:54                                  Agent->requestInventory()
  
[2022-02-16 17:38:28] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:663                                  Agent->requestAgent()
  ajax/agent.php:54                                  Agent->requestInventory()
  
[2022-02-16 17:38:30] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:663                                  Agent->requestAgent()
  ajax/agent.php:54                                  Agent->requestInventory()
  
[2022-02-16 17:38:31] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:34] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:36] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  
[2022-02-16 17:38:38] glpiphplog.NOTICE:   *** PHP Notice (8): Undefined variable: response in /var/www/html/glpi/src/Agent.php at line 625
  Backtrace :
  src/Agent.php:642                                  Agent->requestAgent()
  ajax/agent.php:58                                  Agent->requestStatus()
  


[2022-02-16 18:09:57] glpisqllog.WARNING: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 383
  *** MySQL query warnings:
  SQL: SELECT DISTINCT `glpi_softwares`.`id` AS id, 'username_sa' AS currentuser,
                        `glpi_softwares`.`entities_id`, `glpi_softwares`.`is_recursive`,  `glpi_softwares`.`name` AS `ITEM_Software_1`,
                        `glpi_softwares`.`id` AS `ITEM_Software_1_id`,
                        `glpi_entities`.`completename` AS `ITEM_Software_80`,  `glpi_manufacturers`.`name` AS `ITEM_Software_23`,   GROUP_CONCAT(DISTINCT CONCAT(IFNULL(`glpi_softwareversions`.`name`, '__NULL__'),
                                               '$#$',`glpi_softwareversions`.`id`) ORDER BY `glpi_softwareversions`.`id` SEPARATOR '$$##$$')
                              AS `ITEM_Software_5`,
                  
                   GROUP_CONCAT(DISTINCT CONCAT(IFNULL(`glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`name`, '__NULL__'),
                                               '$#$',`glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id`) ORDER BY `glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id` SEPARATOR '$$##$$')
                              AS `ITEM_Software_4`,
                  
                   COUNT(DISTINCT `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`id`) AS `ITEM_Software_72`,
                      FLOOR(SUM(`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`number`)
                              * COUNT(DISTINCT `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`id`)
                              / COUNT(`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`id`)) AS `ITEM_Software_163`,
                        MIN(`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`number`) AS `ITEM_Software_163_min` FROM `glpi_softwares`LEFT JOIN `glpi_entities` 
                                          ON (`glpi_softwares`.`entities_id` = `glpi_entities`.`id`
                                              )LEFT JOIN `glpi_manufacturers` 
                                          ON (`glpi_softwares`.`manufacturers_id` = `glpi_manufacturers`.`id`
                                              ) LEFT JOIN `glpi_softwareversions` 
                                             ON (`glpi_softwares`.`id` = `glpi_softwareversions`.`softwares_id`
                                                 )LEFT JOIN `glpi_operatingsystems`  AS `glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`
                                          ON (`glpi_softwareversions`.`operatingsystems_id` = `glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id`
                                              ) LEFT JOIN `glpi_items_softwareversions`  AS `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`
                                             ON (`glpi_softwareversions`.`id` = `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`softwareversions_id`
                                                  AND `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_deleted_item` = '0' AND `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_deleted` = '0' AND `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_template_item` = '0' ) LEFT JOIN `glpi_softwarelicenses`  AS `glpi_softwarelicenses_6427cab6934982da77f819e079502198`
                                             ON (`glpi_softwares`.`id` = `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`softwares_id`
                                                  AND `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`is_template` = '0' AND ((`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`expire` IS NULL) OR (`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`expire` > NOW())) ) WHERE  `glpi_softwares`.`is_deleted` = 0  AND `glpi_softwares`.`is_template` = 0  AND (  (   (`glpi_softwares`.`name`  LIKE '%package%'  )  OR  (`glpi_entities`.`completename`  LIKE '%package%'  )  OR  (`glpi_manufacturers`.`name`  LIKE '%package%'  )  OR  (`glpi_softwareversions`.`name`  LIKE '%package%'  )  OR  (`glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`name`  LIKE '%package%'  ))  ) GROUP BY `glpi_softwares`.`id` ORDER BY `id` 
  Warnings: 
1260: Row 942 was cut by GROUP_CONCAT()
  Backtrace :
  src/Search.php:1378                                
  src/Search.php:371                                 Search::constructData()
  src/Search.php:123                                 Search::getDatas()
  src/Search.php:100                                 Search::showList()
  front/software.php:40                              Search::show()
  {"user":"9@glpi.localhost.ru"} 
[2022-02-16 18:15:11] glpisqllog.WARNING: DBmysql::query() in /var/www/html/glpi/src/DBmysql.php line 383
  *** MySQL query warnings:
  SQL: SELECT DISTINCT `glpi_softwares`.`id` AS id, 'username_sa' AS currentuser,
                        `glpi_softwares`.`entities_id`, `glpi_softwares`.`is_recursive`,  `glpi_softwares`.`name` AS `ITEM_Software_1`,
                        `glpi_softwares`.`id` AS `ITEM_Software_1_id`,
                        `glpi_entities`.`completename` AS `ITEM_Software_80`,  `glpi_manufacturers`.`name` AS `ITEM_Software_23`,   GROUP_CONCAT(DISTINCT CONCAT(IFNULL(`glpi_softwareversions`.`name`, '__NULL__'),
                                               '$#$',`glpi_softwareversions`.`id`) ORDER BY `glpi_softwareversions`.`id` SEPARATOR '$$##$$')
                              AS `ITEM_Software_5`,
                  
                   GROUP_CONCAT(DISTINCT CONCAT(IFNULL(`glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`name`, '__NULL__'),
                                               '$#$',`glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id`) ORDER BY `glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id` SEPARATOR '$$##$$')
                              AS `ITEM_Software_4`,
                  
                   COUNT(DISTINCT `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`id`) AS `ITEM_Software_72`,
                      FLOOR(SUM(`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`number`)
                              * COUNT(DISTINCT `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`id`)
                              / COUNT(`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`id`)) AS `ITEM_Software_163`,
                        MIN(`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`number`) AS `ITEM_Software_163_min` FROM `glpi_softwares`LEFT JOIN `glpi_entities` 
                                          ON (`glpi_softwares`.`entities_id` = `glpi_entities`.`id`
                                              )LEFT JOIN `glpi_manufacturers` 
                                          ON (`glpi_softwares`.`manufacturers_id` = `glpi_manufacturers`.`id`
                                              ) LEFT JOIN `glpi_softwareversions` 
                                             ON (`glpi_softwares`.`id` = `glpi_softwareversions`.`softwares_id`
                                                 )LEFT JOIN `glpi_operatingsystems`  AS `glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`
                                          ON (`glpi_softwareversions`.`operatingsystems_id` = `glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`id`
                                              ) LEFT JOIN `glpi_items_softwareversions`  AS `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`
                                             ON (`glpi_softwareversions`.`id` = `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`softwareversions_id`
                                                  AND `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_deleted_item` = '0' AND `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_deleted` = '0' AND `glpi_items_softwareversions_37010ce8f4633da91ded3e0a4c256dc9`.`is_template_item` = '0' ) LEFT JOIN `glpi_softwarelicenses`  AS `glpi_softwarelicenses_6427cab6934982da77f819e079502198`
                                             ON (`glpi_softwares`.`id` = `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`softwares_id`
                                                  AND `glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`is_template` = '0' AND ((`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`expire` IS NULL) OR (`glpi_softwarelicenses_6427cab6934982da77f819e079502198`.`expire` > NOW())) ) WHERE  `glpi_softwares`.`is_deleted` = 0  AND `glpi_softwares`.`is_template` = 0  AND (  (   (`glpi_softwares`.`name`  LIKE '%package%'  )  OR  (`glpi_entities`.`completename`  LIKE '%package%'  )  OR  (`glpi_manufacturers`.`name`  LIKE '%package%'  )  OR  (`glpi_softwareversions`.`name`  LIKE '%package%'  )  OR  (`glpi_operatingsystems_0a35c270152be19b5c8a485502badcd7`.`name`  LIKE '%package%'  ))  ) GROUP BY `glpi_softwares`.`id` ORDER BY `id` 
  Warnings: 
1260: Row 942 was cut by GROUP_CONCAT()
  Backtrace :
  src/Search.php:1378                                
  src/Search.php:371                                 Search::constructData()
  src/Search.php:123                                 Search::getDatas()
  src/Search.php:100                                 Search::showList()
  front/software.php:40                              Search::show()
  {"user":"9@glpi.localhost.ru"}
```

### Your GLPI setup information

<details><summary>Information about system installation and configuration</summary><pre>GLPI 10.0.0-rc1 ( => /var/www/html/glpi)
Installation mode: TARBALL
Current language:en_GB

</details><details><summary>Server</summary><pre> 
Operating system: Linux glpi.localhost.ru 5.4.17-2136.302.7.2.3.el8uek.x86_64 #2 SMP Tue Feb 8 11:17:17 PST 2022 x86_64
PHP 7.4.27 fpm-fcgi (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apcu, bz2, calendar, cgi-fcgi, ctype, curl,
	date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, intl, json, ldap, libxml, mbstring, mysqli, mysqlnd, openssl,
	pcre, pdo_mysql, pdo_sqlite, session, sockets, sodium, sqlite3, standard, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl,
	zip, zlib)
Setup: max_execution_time="30" memory_limit="256M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.37 (Oracle Linux) OpenSSL/1.1.1k ()
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0
Server Software: MariaDB Server
	Server Version: 10.3.28-MariaDB
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: root@localhost/GLPI
	Host info: Localhost via UNIX socket
	
PHP version (7.4.27) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
Database engine version (10.3.28) is supported.
The log file has been created successfully.
Write access to /var/www/html/glpi/files/_cache has been validated.
Write access to /var/www/html/glpi/config has been validated.
Write access to /var/www/html/glpi/files/_cron has been validated.
Write access to /var/www/html/glpi/files has been validated.
Write access to /var/www/html/glpi/files/_dumps has been validated.
Write access to /var/www/html/glpi/files/_graphs has been validated.
Write access to /var/www/html/glpi/files/_lock has been validated.
Write access to /var/www/html/glpi/files/_pictures has been validated.
Write access to /var/www/html/glpi/files/_plugins has been validated.
Write access to /var/www/html/glpi/files/_rss has been validated.
Write access to /var/www/html/glpi/files/_sessions has been validated.
Write access to /var/www/html/glpi/files/_tmp has been validated.
Write access to /var/www/html/glpi/files/_uploads has been validated.
Web access to the files directory should not be allowed but this cannot be checked automatically on this instance.
Make sure access to error log file (/files/_log/php-errors.log) is forbidden; otherwise review .htaccess file and web server configuration.
For security reasons, SELinux mode should be Enforcing.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
zip extension is installed.
bz2 extension is installed.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/html/glpi/marketplace has been validated.
Timezones seems loaded in database.

</details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: /var/www/html/glpi
GLPI_CONFIG_DIR: /var/www/html/glpi/config
GLPI_VAR_DIR: /var/www/html/glpi/files
GLPI_MARKETPLACE_DIR: /var/www/html/glpi/marketplace
GLPI_USE_CSRF_CHECK: 1
GLPI_CSRF_EXPIRES: 7200
GLPI_CSRF_MAX_TOKENS: 100
GLPI_USE_IDOR_CHECK: 1
GLPI_IDOR_EXPIRES: 7200
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: 
GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org
GLPI_INSTALL_MODE: TARBALL
GLPI_NETWORK_MAIL: glpi@teclib.com
GLPI_NETWORK_SERVICES: https://services.glpi-network.com
GLPI_MARKETPLACE_ALLOW_OVERRIDE: 1
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: 1
GLPI_USER_AGENT_EXTRA_COMMENTS: 
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: 1
GLPI_AJAX_DASHBOARD: 1
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: 0
GLPI_CENTRAL_WARNINGS: 1
GLPI_DOC_DIR: /var/www/html/glpi/files
GLPI_CACHE_DIR: /var/www/html/glpi/files/_cache
GLPI_CRON_DIR: /var/www/html/glpi/files/_cron
GLPI_DUMP_DIR: /var/www/html/glpi/files/_dumps
GLPI_GRAPH_DIR: /var/www/html/glpi/files/_graphs
GLPI_LOCAL_I18N_DIR: /var/www/html/glpi/files/_locales
GLPI_LOCK_DIR: /var/www/html/glpi/files/_lock
GLPI_LOG_DIR: /var/www/html/glpi/files/_log
GLPI_PICTURE_DIR: /var/www/html/glpi/files/_pictures
GLPI_PLUGIN_DOC_DIR: /var/www/html/glpi/files/_plugins
GLPI_RSS_DIR: /var/www/html/glpi/files/_rss
GLPI_SESSION_DIR: /var/www/html/glpi/files/_sessions
GLPI_TMP_DIR: /var/www/html/glpi/files/_tmp
GLPI_UPLOAD_DIR: /var/www/html/glpi/files/_uploads
GLPI_INVENTORY_DIR: /var/www/html/glpi/files/_inventories
GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/
GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/
GLPI_I18N_DIR: /var/www/html/glpi/locales
GLPI_VERSION: 10.0.0-rc1
GLPI_SCHEMA_VERSION: 10.0.0-rc1@4f12cc11183dfb486b84e28e8dad44a9259736c3
GLPI_MARKETPLACE_PRERELEASES: 1
GLPI_MIN_PHP: 7.4.0
GLPI_MAX_PHP: 8.2.0
GLPI_YEAR: 2022

</details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.6 in (/var/www/html/glpi/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.5.3 in (/var/www/html/glpi/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/var/www/html/glpi/vendor/simplepie/simplepie/library)
mpdf/mpdf in (/var/www/html/glpi/vendor/mpdf/mpdf/src)
michelf/php-markdown in (/var/www/html/glpi/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/html/glpi/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/html/glpi/vendor/iamcal/lib_autolink)
sabre/dav in (/var/www/html/glpi/vendor/sabre/dav/lib/DAV)
sabre/http in (/var/www/html/glpi/vendor/sabre/http/lib)
sabre/uri in (/var/www/html/glpi/vendor/sabre/uri/lib)
sabre/vobject in (/var/www/html/glpi/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/var/www/html/glpi/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/var/www/html/glpi/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/var/www/html/glpi/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/html/glpi/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/var/www/html/glpi/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/var/www/html/glpi/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/html/glpi/vendor/symfony/console)
scssphp/scssphp in (/var/www/html/glpi/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/var/www/html/glpi/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/var/www/html/glpi/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/var/www/html/glpi/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/var/www/html/glpi/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/var/www/html/glpi/vendor/ramsey/uuid/src)
psr/log in (/var/www/html/glpi/vendor/psr/log/Psr/Log)
psr/simple-cache in (/var/www/html/glpi/vendor/psr/simple-cache/src)
psr/cache in (/var/www/html/glpi/vendor/psr/cache/src)
league/csv in (/var/www/html/glpi/vendor/league/csv/src)
mexitek/phpcolors in (/var/www/html/glpi/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/var/www/html/glpi/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/var/www/html/glpi/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/var/www/html/glpi/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/var/www/html/glpi/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/var/www/html/glpi/vendor/paragonie/sodium_compat/src)
symfony/cache in (/var/www/html/glpi/vendor/symfony/cache)
html2text/html2text in (/var/www/html/glpi/vendor/html2text/html2text/src)
symfony/dom-crawler in (/var/www/html/glpi/vendor/symfony/dom-crawler)
twig/twig in (/var/www/html/glpi/vendor/twig/twig/src)
twig/string-extra in (/var/www/html/glpi/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 in (/var/www/html/glpi/vendor/symfony/polyfill-php80)

</details><details><summary>LDAP directories</summary><pre> 
Server: '192.168.1.3', Port: '3268', BaseDN: 'DC=localhost,DC=ru', Connection filter:
		'(&(objectClass=user)(objectCategory=person)(!(userAccountControl:1.2.840.113556.1.4.803:=2)))', RootDN:
		'CN=ldapsearch-glpi,OU=Special,DC=localhost,DC=ru', Use TLS: none

</details><details><summary>SQL replicas</summary><pre> 
Not active

</details><details><summary>Notifications</summary><pre> 
Way of sending emails: SMTP (anonymous@mail2.localhost.ru)

</details><details><summary>Plugins list</summary><pre> 
	connections          Name: Connections                    Version: 10.0.0-rc2 State: Installed / not activated
	manageentities       Name: Entities portal                Version: 4.0.0-rc1  State: Installed / not activated
	glpiinventory        Name: GLPI Inventory                 Version: 1.0.0-rc4  State: Enabled
	addressing           Name: IP Adressing                   Version: 3.0.0-rc2  State: Installed / not activated
	mreporting           Name: More Reporting                 Version: 1.8.0-rc2  State: Enabled
	tag                  Name: Tag Management                 Version: 2.9.0-rc3  State: Installed / not activated
	useditemsexport      Name: Used items export              Version: 2.5.0-rc2  State: Enabled
	shellcommands        Name: Служебные команды        Version: 4.0.0-rc2  State: Enabled
	typology             Name: Типологий                      Version: 3.0.0-rc1  State: Installed / not activated

</pre></details>

**Comment by Badkempachi** [comment]
![1](https://user-images.githubusercontent.com/97535165/154301914-5f25fc08-b0d7-4bef-8391-5fb8aa2b8d37.png)
![2](https://user-images.githubusercontent.com/97535165/154301920-a6449c89-a3f0-4b1d-bae1-ea9442fe3d81.png)
![3](https://user-images.githubusercontent.com/97535165/154301921-a886eb67-32be-4153-b51e-7db3c1c80053.png)

**Comment by cconard96** [comment]
Can you re-test with the latest release or at least the fix from #10693?

**Comment by Badkempachi** [comment]
> Can you re-test with the latest release or at least the fix from #10693?

Hi, @cconard96 
Yes, I re-tested on the last nightly build, the information is displayed correctly and warnings, errors that were previously, are now absent.
Thanks :)
